packages/webassembly-playground/webpack.config.js

```diff
@@ -77,6 +77,9 @@ const baseConfig = {
       },
     },
   },
+  experiments: {
+    asyncWebAssembly: true,
```

Comment:
Why do we need to set this option? Frankly, I don't know what webpack's `experiments.asyncWebAssembly` does exactly. Quote from webpack's doc:
> Support the new WebAssembly according to the [updated specification](https://github.com/WebAssembly/esm-integration), it makes a WebAssembly module an async module.

But it looks like `wasm-pack` uses just `WebAssembly.instantiateStreaming` or `.instantiate`, which were introduced before wasm's esm-integration proposal.

Reply:
I'm not sure, but you can try removing this and check if it works.